reference/constraints/Bic.rst

```diff
@@ -131,4 +133,20 @@ For example, if you want to compare the ``$bic`` property of some object
 with regard to the ``$iban`` property of the same object, use
 ``propertyPath="iban"`` in the comparison constraint of ``$bic``.
 
+allowLowerCase
+~~~~~~~~~~~~~~~~
+
+**type**: ``boolean`` **default**: false
+
+If this option is set to ``false`` and value contains a lower letter, a validation
```

Comment:
Lowercase letter.... instead of lower letter... ?

Reply:
The default is false and the explanation is using false too? Shouldn’t you explain what happens when setting this option to true?

I am on a phone right now... or is it the same for other options?

Reply:
I based myself on collection constraint documentation. allowMissingFields is to false by default and the explanation is using false too. But indeed I will append case when this options is to true